circle.yml

```diff
@@ -15,3 +15,9 @@ test:
   override:
     - make check-version
     - make test-ci
+
+deployment:
+  hub:
+    tag: /[0-9]+\.[0-9]+\.[0-9]+/
+    commands:
+      - make add-binary-as-artifact -e CIRCLE_ARTIFACTS=$CIRCLE_ARTIFACTS
```

Comment:
Do you not have to build the go binary in a separate make command?

If so, this make command would sit well within the `test` block.